<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSaveScreen</class>
 <widget class="QDialog" name="DialogSaveScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>226</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QPushButton" name="BtnOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="BtnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Legend position:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QPushButton" name="BtnFont">
        <property name="text">
         <string>Set font</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="RadioRB">
        <property name="text">
         <string>right bottom</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="RadioRT">
        <property name="text">
         <string>right top</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="RadioNone">
        <property name="text">
         <string>no legend</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="RadioLB">
        <property name="text">
         <string>left bottom</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="RadioLT">
        <property name="text">
         <string>left top</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="CheckRootGroups">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When checked root groups are used to generate legend no mather if any data from group will be visible on output image. Unchecked will cause using inly groups, that are direct parent of visible items. Using unchecked variant may be significantly slower.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Use root groups</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string>Image size:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="SpinWidth">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8192</number>
        </property>
        <property name="value">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="SpinHeight">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>8192</number>
        </property>
        <property name="value">
         <number>768</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BtnOk</sender>
   <signal>clicked()</signal>
   <receiver>DialogSaveScreen</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogSaveScreen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnFont</sender>
   <signal>clicked()</signal>
   <receiver>DialogSaveScreen</receiver>
   <slot>slSetFont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slSetFont()</slot>
 </slots>
</ui>
